provisioning/service/tests/Config/AttestationMechanismTests.cs

```diff
@@ -11,21 +11,218 @@ namespace Microsoft.Azure.Devices.Provisioning.Service.Test
     [TestClass]
     public class AttestationMechanismTests
     {
-        /* SRS_ATTESTATION_MECHANISM_21_001: [The constructor shall throw ArgumentException if the provided attestation is null.] */
-        /* SRS_ATTESTATION_MECHANISM_21_002: [If the provided attestation is instance of TpmAttestation, the constructor shall store the provided tpm keys.] */
+        private const string KEY = "AToAAQALAAMAsgAgg3GXZ0SEs/gakMyNRqXXJP1S124GUgtk8qHaGzMUaaoABgCAAEMAEAgAAAAAAAEAxsj" +
```

Comment:
Nit: naming should be `Key`, not all caps.

Reply:
Done